esp32_marauder/CommandLine.cpp

```diff
@@ -9,6 +9,9 @@ void CommandLine::RunSetup() {
   Serial.println(F("\n\n--------------------------------\n"));
   Serial.println(F("         ESP32 Marauder      \n"));
   Serial.println("            " + version_number + "\n");
+  #ifdef WRITE_PACKETS_SERIAL
```

Comment:
same as #253 because I had merged that branch locally before this push

Reply:
> If the changes from the previous MRs you've submitted are contained in this MR, would you mind closing those. I will run the workflows once you do that and I will review the changes.

Yes closed 1 duplicate now. The BT messages MR #252 stands on its own and isn't duplicated here because it was originally done on `master` and has no conflicts with current develop.

Thanks